core/src/network/direct_message.rs

```diff
@@ -0,0 +1,21 @@
+use holochain_core_types::{cas::content::Address, validation::ValidationPackage};
+
+/// These are the different kind of node-to-node messages
```

Comment:
I think we should add "low-level system" so as not to confuse the App level node-to-node messaging.